src/assets/scss/forms.scss

```diff
@@ -1,61 +1,3 @@
-.c-custom-select {
-    -moz-appearance: none;
-    -webkit-appearance: none;
-    appearance: none;
-    box-sizing: border-box;
-    display: block;
-    width: 100%;
-    max-width: 100%;
-    min-width: 0;
-    padding: 0.625rem 0.875rem;
-    padding-right: calc(0.875rem * 3);
-    padding-inline-end: calc(0.875rem * 3);
-    font: inherit;
-    color: var(--body-text-color);
-    color: inherit;
-    line-height: 1.3;
-    border: 1px solid var(--border-color);
-    border-radius: var(--border-radius);
-    box-shadow: var(--shadow-xs);
-    background-color: var(--body-background-color);
-    background-image:
-        url("data:image/svg+xml,%3Csvg width='20' height='21' viewBox='0 0 20 21' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M5 7.60938L10 12.6094L15 7.60938' stroke='%23667085' stroke-width='1.66667' stroke-linecap='round' stroke-linejoin='round'/%3E%3C/svg%3E%0A"),
-        linear-gradient(
-            to bottom,
-            var(--body-background-color) 0%,
-            var(--body-background-color) 100%
-        );
-    background-repeat: no-repeat, repeat;
-    background-position: right calc(0.875rem * 1.5) top 50%, 0 0;
-    background-size: 1em auto, 100%;
-}
-
-.label__text.label__text {
-    display: flex;
-    align-items: center;
-    gap: 0.5rem;
-    font-size: 0.875rem;
-    font-family: var(--text-font);
-    color: inherit;
```

Comment:
There is a difference from the reserved line 121, which is `color: var(--color-neutral-700);`, but I think the reserved one is the expected behavior.

Reply:
Yea this was getting over written by the above one so removed it